src/django/api/fixtures/contributors.json

```diff
@@ -1,7 +1,7 @@
 [
     {
         "model": "api.contributor",
-        "pk": 2,
+        "pk": 1,
```

Comment:
Just curious about some context here -  is `pk` the user ID and `fields.admin` being the contributor ID?

Reply:
`pk` stands for "primary key". Since it's on the api.contributor model, it's the primary key / id for the contributor. The admin id is the id for the user who administers the account (so the user id). Hope that makes sense!